api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupQualityImprovementReport20.java

```diff
@@ -76,7 +77,7 @@ public ReportDefinition constructReportDefinition() {
       reportDesign =
           createXlsReportDesign(
               reportDefinition,
-              "Melhoria de Qualidade - 2020.xls",
+              "Template_MQ_2022.xls",
```

Comment:
This it the name that will show-up in the generated report, I think is much better to name it Melhoria de Qualidade - 2022.xls

Reply:
It's a template name, the generated report name is "Melhoria de Qualidade - 2022"